app/views/components/h1_heading_with_fieldset.scala.html

```diff
@@ -0,0 +1,27 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import ui.H1
+
+@( h1: H1 )( implicit messages: Messages, ctx: PageContext)
+<legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
```

Comment:
xl seems on the large size. What is your reasoning?

Reply:
I see question uses xl so I guess it is consistent.

Reply:
It matched the input with label component

Reply:
have a look at h1_heading.scala.html. There is a setting on the page which determines if the reduced heading sizes should be used. something like class="@if(ctx.page.reduceHeadings){govuk-heading-l} else {govuk-heading-xl}"

Reply:
also applies to h1_heading_with_label.scala.html